cli/azd/cmd/util_test.go

```diff
@@ -105,7 +106,11 @@ func Test_promptEnvironmentName(t *testing.T) {
 			}, nil
 		})
 
-		resourceManager := infra.NewAzureResourceManager(*mockContext.Context)
+		azCli := azcli.NewAzCli(mockContext.Credentials, azcli.NewAzCliArgs{
+			HttpClient: mockContext.HttpClient,
+		})
```

Comment:
Should this use the mock package instead?

Reply:
Indeed - it should.